### ISSUE TITLE
Currently, it appears that you are just supporting localhost; however, we require support for the healenium backend hosted on a remote server.

### ISSUE THREAD

**Issue by antonjerald** [healenium/healenium-idea#23]
Currently, it appears that you are just supporting localhost; however, we require support for the healenium backend hosted on a remote server.

With regards to Intellj idea plugin, are you supporting only local host at the moment. It would be awesome if you add support for remote servers.

**Comment by Aliaksei-Ashukha** [comment]
Yes, currently plugin support only localhost. We are going to support remote servers as well.

**Comment by antonjerald** [comment]
Ok Thank you.
